meshery

```diff
@@ -62,16 +62,15 @@ unzip ${PWD}/meshery.zip
 
 printf "\nInstalling mesheryctl in /usr/local/bin.\n"
 WHOAMI=$(whoami)
-if [ $OSEXT = "Linux" ] ; then
```

Comment:
Is there a need to remove the `OSEXT` check?

Reply:
@kushthedude great question. Here's additional context to consider - https://github.com/layer5io/meshery/pull/674#discussion_r379044558

Reply:
@kushthedude after going through #674 if you still face any issue let me know.